source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -29,8 +29,42 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+- Most search functionality included in the core platform now supports Meilisearch as an alternative to Elasticsearch. For instances deployed using Tutor, only Meilisearch is supported, and Tutor will automatically provision Meilisearch and create the indexes during the upgrade.
```

Comment:
>  For instances deployed using Tutor, only Meilisearch is supported

Is this a breaking change?

Reply:
I'm not sure what it would "break". The related functionality will continue to work more or less the same, just using a different backend. I guess if they are running some additional app/module that expects to piggyback the platform's ElasticSearch index, operators would now need to deploy Elasticsearch separately but I don't think that's really an officially supported use case anyways.

Reply:
If you are upgrading from an old Tutor instance pre-Meilisearch, (say, Olive), will you be forced to convert to Meilisearch?

Reply:
Edit: I guess it's not breaking, but it's a bit not clear to me that ES will be dropped and I can't use it so maybe a tad bit more explicit would help.